src/Functions/FunctionSqid.cpp

```diff
@@ -1,104 +0,0 @@
-#include "config.h"
```

Comment:
I renamed this file in [another PR](https://github.com/ClickHouse/ClickHouse/pull/58517) ... weird why it is shown as deleted here.